src/poparser.js

```diff
@@ -14,18 +14,18 @@ export function parse (input, options = {}) {
   const parser = new Parser(input, options);
 
   return parser.parse();
-};
+}
 
 /**
  * Parses a PO stream, emits translation table in object mode
  *
- * @typedef {{ defaultCharset: strubg, validation: boolean }} Options
+ * @typedef {{ defaultCharset: string, validation: boolean }} Options
```

Comment:
i made some cosmetical changes like this because the type checking wouldn't work without

Reply:
Yes, that's great! I would suggest to include them in the pre-TS PR.